week2/homework/src/add.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+function add(todoTask) {
+  if (todoTask === undefined) {
+    console.log('Please fill in a legitimate task to add');
+  } else {
+    fs.appendFile('./data.txt', `${todoTask}\n`, error => {
+      if (error) throw error;
+    });
+    console.log(`${todoTask} added to the list`);
```

Comment:
This should probably be inside the callback

Reply:
Yes you are right I removed it